src/Core/Cache/CacheFactory.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace SilverStripe\Core\Cache;
+
+use Psr\SimpleCache\CacheInterface;
+
+interface CacheFactory
```

Comment:
Sorry, I revoke that. CacheFactory just needs to extend Factory :) Having `CacheFactory` as a service name is useful.